src/steps/10-FinancialDetails/GInvoiceLearnMore.vue

```diff
@@ -78,6 +78,6 @@ import { Component } from "vue-property-decorator";
 export default class GInvoiceLearnMore extends Vue {
   private fiscalServiceHref = "https://www.fiscal.treasury.gov/g-invoice/index.html";
   private gInvoiceLoginHref =
-   " https://www.igt.fiscal.treasury.gov";
+    " https://www.igt.fiscal.treasury.gov";
 }
```

Comment:
- [ ] pls remove space before the `https` as a safety measure to ensure that space never gets encoded

Reply:
@tonysumner-ccpo The linting updated the whitespace before the `"`, not inside the string, and this has been in the codebase for almost a year now.  Ill create a PR to fix this though